### DISCUSSION TITLE
训练的模型目录不包含phone_id_map等文件

### DISCUSSION THREAD

**Discussion by jerryuhoo** [PaddlePaddle/PaddleSpeech#1367] [Q&A]
之前我自己训练的时候有多个模型，有时候会忘了存推理要用的文件就把dump删了，我感觉把所有的phone_id_map.txt，speaker_id_map.txt、tone_id_map.txt、feats_stats这些文件都拷贝一份，放到训练生成的目录下比较方便整洁，比如说在train.sh这个脚本之前加入自动复制，不过我不知道最近的代码里是否加入了这个功能。我没仔细看，如果已经有的话麻烦告诉我一下在哪里，谢啦

**Comment by yt605155624** [comment]
没有，目前只是在 train.py 里面增加了拷贝 default.yaml 的功能 https://github.com/PaddlePaddle/PaddleSpeech/blob/49fd55dc16d60bc830c728752c50d3845e016386/paddlespeech/t2s/exps/fastspeech2/train.py#L145
